sos/report/plugins/kubernetes.py

```diff
@@ -247,62 +264,76 @@ def postproc(self):
         # output that is not hit by the previous iteration.
         self.do_cmd_private_sub(self.kube_cmd)
 
+        pathexp_config_files = fr'^({"|".join(self.config_files)})'
+        self.do_file_private_sub(pathexp_config_files)
+
+        # clear base64 encoded PEM from kubeconfigs files
+        regexp = r'LS0tLS1CRUdJ[A-Za-z0-9+/=]+'
+        subst = '***** SCRUBBED BASE64 PEM *****'
+
+        self.do_path_regex_sub(pathexp_config_files, regexp, subst)
+
+        pathexp_files = fr'^({"|".join(self.files)})'
+        self.do_path_regex_sub(pathexp_files, regexp, subst)
```

Comment:
Can§t we merge this into one `do_path_regexp_sub`? Or would it be too confused?

Reply:
done